backend/src/routes/routes.ts

```diff
@@ -1,19 +1,19 @@
-import express, { Request, Response } from "express";
-import { User } from "../models/users";
+// import express, { Request, Response } from "express";
```

Comment:
will we need this commented out code in future?

Reply:
so right, we don't need this anymore